src/main/kotlin/no/nav/familie/ba/sak/common/RessursUtils.kt

```diff
@@ -40,7 +40,7 @@ object RessursUtils {
         return ResponseEntity.status(HttpStatus.FORBIDDEN)
             .body(
                 Ressurs.ikkeTilgang<T>(rolleTilgangskontrollFeil.melding)
-                    .copy(frontendFeilmelding = "Mangler tilgang til opplysningene")
+                    .copy(frontendFeilmelding = rolleTilgangskontrollFeil.melding.ifBlank { "Mangler tilgang til opplysningene" })
```

Comment:
```suggestion
                    .copy(frontendFeilmelding = rolleTilgangskontrollFeil.frontendFeilmelding.ifBlank { "Mangler tilgang" })
```

Vi vet ikke om det er henting eller skriving vi skal gjøre :) Og vi bør bare hente fra frontendFeilmelding